apps/aecontract/src/aect_sophia.erl

```diff
@@ -75,7 +76,7 @@ simple_call(Code, Function, Argument) ->
                     , gasPrice => 1
                     , origin => 0
                     , value => Amount
-                    , currentCoinbase => 1
+                    , currentCoinbase => 0
```

Comment:
currentCoinbase in aect_evm:call (line 46) may need review (it is 1 now).

Reply:
Tracked in #160281213